webComponent.html

```diff
@@ -2,32 +2,31 @@
 <html>
   <head>
     <script>
-
-      window.changeAttr = function(attr, val) {
+      window.changeAttr = function (attr, val) {
```

Comment:
I`m wondering why this files exists ? and why it's not in public folder? Maybe we can remove it.

Reply:
Yeah sure, I purged it